lib/spack/spack/cmd/install.py

```diff
@@ -48,6 +48,9 @@ def setup_parser(subparser):
         '-n', '--no-checksum', action='store_true', dest='no_checksum',
         help="Do not check packages against checksum")
     subparser.add_argument(
+        '-p', '--parallel', action='store_true', dest='build_parallel',
```

Comment:
@tgamblin - but this `-j<number>` is applied for every job independently. As there is no master make process they spawn from.